src/shogun/evaluation/Calibration.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Heiko Strathmann, Dhruv Arya
+ */
+
+#ifndef _CALIBRATION_H__
+#define _CALIBRATION_H__
+
+#include <shogun/lib/config.h>
+
+#include <shogun/machine/Machine.h>
+
+namespace shogun
+{
+	/** @brief Base class for all calibration methods. Call fit to 
+	fit the parameters on the predictions and the true labels. Call calibrate to calibrate predictions. **/
+	class CCalibration : public CSGObject
+	{
+	public:
+		/** Constructor. */
+		CCalibration()
+		{
+
+		}
+		/** Destructor. */
+		virtual ~CCalibration()
+		{
+			
+		}
+
+		virtual const char* get_name() const
+		{
+			return "Calibration";
+		}
+
+		/** Fit calibration parameters for binary labels.
+		* @param predictions The predictions outputted by the machine
+		* @param targets The true labels corresponding to the predictions
+		* @return boolean indicating whether the calibration was succesful
```

Comment:
minor: no need to state the type in return, c++ tell you that already

Reply:
Understood.